flexmeasures/data/models/audit_log.py

```diff
@@ -178,3 +185,33 @@ def add_record(
             affected_asset_id=asset.id,
         )
         db.session.add(audit_log)
+
+
+def truncate_event(
+    event: str, attr_key: str, old_value: str, new_value: str, max_length: int = 250
+) -> str:
+    """Truncate the event string and place ellipses in the middle if it exceeds max_length."""
+
+    # Truncate values only if event length exceeds max_length
+    if len(event) > max_length:
+        truncated_new_value = truncate_string(str(new_value), 60)
+        truncated_old_value = truncate_string(str(old_value), 60)
+    else:
+        truncated_new_value = new_value
+        truncated_old_value = old_value
+
+    # Construct the log entry
+    change_summary = (
+        f"Attr: {attr_key}, From: {truncated_old_value}, To: {truncated_new_value}"
+    )
+
+    # Ensure the entire log entry fits within the maximum length
```

Comment:
Technically, the event is not the "entire log entry" so I guess the comment needs a slight update.

Maybe the whole function is actually truncating the change summary, not the event...

Reply:
yes in this case this function `truncate_event` is truncating the event  `f"Attr : {attr_key}, From: {truncated_old_value}, To: {truncated_new_value}"` so that comment needs to be changed to address that.